src/onegov/landsgemeinde/models/agenda.py

```diff
@@ -149,3 +149,21 @@ def title_parts(self) -> list[str]:
             for line in (self.title or '').splitlines()
             if (stripped_line := line.strip())
         ]
+
+    @property
+    def more_files(self) -> list[LandsgemeindeFile]:
+        files = self.files
+        if self.memorial_pdf:
+            return [
+                file for file in files
+                if file != self.memorial_pdf and isinstance(file,
+                                                            LandsgemeindeFile)
```

Comment:
```suggestion
                if file != self.memorial_pdf
                if isinstance(file, LandsgemeindeFile)
```

minor style nit. Comprehension may contain an arbitrary number of conditions, so using `and` is not necessary.

Although I'm not sure if the second condition is even necessary. If it is, you should probably check it in both branches. It doesn't really make sense to only check it if `memorial_pdf` exists.

It seems easier however to just change all the annotations from `LandsgemeindeFile` to `File` though. If you really wanted "type safety" (it could still be circumvented in the database itself), ensuring that only `LandsgmeindeFile` can be linked to `Assembly`, you should change the annotation of `Assembly.files` to `relationship[list[LandsgemeindeFile]]` and ignore the type error there.

It also seems more efficient to collapse the two branches by doing this:

```python
return [file for file in files if file.name != 'memorial_pdf']
```